src/openrct2-ui/windows/Finances.cpp

```diff
@@ -379,19 +379,19 @@ class FinancesWindow final : public Window
         }
 
         // Expenditure / Income values for each month
-        int16_t currentMonthYear = static_cast<int16_t>(gDateMonthsElapsed);
```

Comment:
Why not just make it a uint16_t all along, rather than casting it at the end?

Reply:
Good point. uint16_t makes whole code a lot simpler